### PR TITLE
register/login is connected with api

### PR DESCRIPTION
**Pull request by asliddintursunov**

### CODE REVIEW

**Comment by asliddintursunov** [comment]
If you remember when I open the website on another device, it was giving an error ( initial articles is undefined ) on Infinite Scroll page. I fixed it using "server action function", it's like a server component like the data will be fetched on the server and can be used in the server as well as the client component.

File: app/editor/components/Editor.tsx [hunk 1]
```diff
@@ -56,12 +67,7 @@ export default function PrimeReactEditor() {
       <Editor
         value={postBody}
         onTextChange={(e: any) => setPostBody(e.htmlValue)}
-        style={{
-          height: "50vh",
-          display: postBody && "flex",
-          justifyContent: postBody && "center",
-          cursor: "text",
-        }}
+        style={editorStyle}
```
Comment (muslimchikkk) [inline app/editor/components/Editor.tsx:70]: How about putting this style into css file and using a classname?

File: app/editor/components/Editor.tsx [hunk 2]
```diff
@@ -19,17 +20,19 @@ export default function PrimeReactEditor() {
...
     };
     axios
       .post(`${baseURL}/create-post`, postData)
-      .then((res) => console.log(res.data))
+      .then((res) => setPost(res.data))
       .catch((err) => console.error(err));
   };
 
   const handleClear = function () {
+    setPostTitle("");
```
Comment (muslimchikkk) [inline app/editor/components/Editor.tsx:34]: Always good to set a variable to null or undefined while initialising.

**Review by muslimchikkk — APPROVED** [review]

File: app/editor/components/Editor.tsx (hunk 1 above)
Comment (asliddintursunov) [reply app/editor/components/Editor.tsx:70]: I'll fix it

**Pull request merged**